.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Thanks for helping us make a better snapd!
+
+Have you signed the [license
+agreement](https://www.ubuntu.com/legal/contributors) and read the [contribution
+guide](https://github.com/snapcore/snapd/blob/master/CONTRIBUTING.md)?
+
+Please make sure that the following items have been verified: (please leave a
+note if one of those is not applicable):
+
+- [ ] there are unit tests covering your change
+- [ ] unit tests are passing (./run-checks --unit)
+- [ ] static files tests are passing (./run-checks --static)
+- [ ] snapd snap build works with latest snapcraft (snapcraft --verbose)
```

Comment:
This is (1) rather time consuming and (2) strictly speaking incomplete as it does not include deb artifices.

There is a balance between  running tests upfront to save time/noise in CI failure VS not duplicating expensive test.